packages/eddsa-poseidon/src/blake.ts

```diff
@@ -1,6 +1,13 @@
 // @ts-ignore
 import { Blake512 } from "blake-hash/lib"
 
+/**
+ * Computes the Blake512 hash of the input message.
+ * Blake512 is a cryptographic hash function that produces a hash value of 512 bits,
+ * commonly used for data integrity checks and other cryptographic applications.
+ * @param message - The input data to hash, provided as a Buffer.
```

Comment:
```suggestion
 * @param message The input data to hash, provided as a Buffer.
```

We're not using the hyphen in the other packages. We should keep it consistent imo.